src/RoadmapTopic/RoadmapTopic.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Lesson } from "models";
+import { RoadmapTopicContainer, RoadmapTopicTitle } from "./RoadmapTopic.styled";
+import { AppColors, Colors } from "styles/types";
+import { capitalizeFirstLetter } from "utils";
+import Popover from "molecules/Popover/Popover";
+import RoadmapLesson from "RoadmapLesson/RoadmapLesson";
+import Button from "atoms/Button/Button";
+
+interface RoadmapTopicProps {
+  name: string;
+  trackName?: string;
+  lessons: Lesson[];
+}
+const RoadmapTopic: React.FC<RoadmapTopicProps> = ({ name, lessons, trackName }) => {
+  return (
+    <RoadmapTopicContainer>
+      <RoadmapTopicTitle>{capitalizeFirstLetter(name)}</RoadmapTopicTitle>
+      {lessons.map((lesson, i) =>
+        lesson.wip ? (
+          <Button
+            key={lesson.id + i}
```

Comment:
if IDs are unique and non-numeric then I'd skip adding `+i` here

Reply:
There are no IDs in the provided data. I added it to the model and copy the name to it (for future improvements to the data), hence the use of index. Plus, these items are not meant to ever be re-arranged or modified.